datacenter/dtr/2.2/guides/admin/upgrade/index.md

```diff
@@ -43,6 +74,16 @@ By default the upgrade command runs in interactive mode and prompts you for
 any necessary information. You can also check the
 [reference documentation](../../../reference/cli/index.md) for other existing flags.
 
+The upgrade command will start replacing every container in your DTR cluster,
+one replica at a time. It will also perform certain data migrations. If anything
+fails or the upgrade is interrupted for any reason, you can re-run the upgrade
+command and it will resume from where it left off.
+
+## Patch upgrade
+
+A patch upgrade changes only the DTR containers and it should be a relatively
+safe operation. The command is the same as for a minor upgrade.
```

Comment:
what is relatively safe? what's the difference with "absolutely" safe? is there any downtime involved at all and, if so, how long does it last?